universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -61,16 +61,18 @@ public QuestionController(
         .thenApplyAsync(
             readOnlyService -> {
               try {
-                QuestionDefinition definition = questionForm.getBuilder().setVersion(1L).build();
-                ErrorAnd<QuestionDefinition, CiviFormError> result = service.create(definition);
-                if (result.isError()) {
-                  String errorMessage = joinErrors(result.getErrors());
+                try {
```

Comment:
Yes, and most of the diff is just due to the tab reformatting. The nested try-catch is necessary because `InvalidQuestionType` can now be thrown by `renderNewQuestionForm`